src/fdk_model_publisher/service/mapper.py

```diff
@@ -72,6 +72,14 @@ def map_model_from_dict(
     model.modelelements = model_element_mapper.extract_model_items(
         partial_model.schema, partial_model.endpoint_description
     )
+    format = FoafDocument()
+    format.identifier = (
+        "http://publications.europa.eu/resource/authority/file-type/JSON"
```

Comment:
Vi bare antar at alle api er json?

Reply:
Ja, det er det som er sakens kjerne her. Alle som går igjennom fdk-model-publisher er i format JSON. Det var det som jeg og Mats kom fram til og Tore mente også at dette var riktig. Så kan vi forandre dette senere dersom fdk-model-publisher skulle støtte andre formater enn JSON. Alle modellene som kommer gjennom herfra er altså p.t. JSON

Reply:
Vi var vel bort i at noen spesifikasjoner var på et yaml-format, eller hva @matsjs?

Reply:
Dette kan vi også fikse @ToHelland . Da blir det kanskje mer riktig å sette dct:format yaml der hvor dette gjøres og så json der hvor det er json?

Reply:
Hvis dere vil så kan jeg endre dette til at det vises yaml når det yaml og json kun når det er json? Er det ok?

Reply:
Om det er informasjon du har tilgjengelig så er jo det bedre

Reply:
Da har jeg avtalt med @ToHelland at vi setter json kun der det er json eksplisitt dvs på json filtype. Der det er yaml gjør vi ikke noe enda inntil vi får sjekket opp om det finnes EU-vokabular for yaml, men dér skal vi altså ikke sette json, for det vil jo selvfølgelig bli feil. Nye oppdtaringer kommer! Takk til alle som spørr og kommenterer fordi her er det ikke nødvendigvis helt klart hvordan dette skal være! :+1:

Reply:
Det kan ofte forekomme at openAPI-spekken er yaml.